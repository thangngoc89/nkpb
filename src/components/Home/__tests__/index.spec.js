import React from 'react'
import renderer from 'react-test-renderer'

jest.mock('../../../containers/Sidebar')
jest.mock('../index.css')

const Home = require('../Home').default

describe('Home', () => {
  it('should render', () => {
    const component = renderer.create(
      <Home />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
