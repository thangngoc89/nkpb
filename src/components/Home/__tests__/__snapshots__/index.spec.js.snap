exports[`Home should render 1`] = `null`;
